### PR TITLE
Node vamc operating status and alerts

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Another day, another transformer.

This one's interesting because it introduced the possibility for circular references.

I also had to add more fields to other content models because they're used in this one (and nowhere else).

## Testing done
Yup.

**Note:** The test data for this can be found in a [separate PR](https://github.com/department-of-veterans-affairs/vets-website/pull/11612) because...there are 252 extra JSON files needed to test this silly thing. Because entity hierarchy.

We mostly throw out the data in all those JSON files because of reasons, but it'd take a heavy reworking of the entity expansion logic to change that up. In my opinion, it's not worth it. Annoying that we have _so many_ "useless" files, but not worth making the entity expansion significantly more complicated (at least the way I'm seeing it in my head now).

## Screenshots
N/A

## Acceptance criteria
- [x] The assemble entity tree unit test passes
  - Well, it would if the test data were there, anyhow...
- [x] `npm run build:content:test` finishes without error

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/schemas/common/entityMetaTags.js [hunk 1]
```diff
@@ -4,9 +4,12 @@ module.exports = {
   items: {
     type: 'object',
     properties: {
+      __typename: { type: 'string' },
       type: { type: 'string' },
       key: { type: 'string' },
       value: { type: 'string' },
     },
+    required: ['key', 'value'],
+    oneOf: [{ required: ['__typename'] }, { required: ['type'] }],
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/schemas/common/entityMetaTags.js]: I like it!

File: src/site/stages/build/process-cms-exports/schemas/transformed/node-full_width_banner_alert.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  type: 'object',
+};
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/schemas/transformed/node-full_width_banner_alert.js]: Should there be more here?

File: src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      },
+    },
+    fieldOffice: {
+      oneOf: [
+        { $ref: 'transformed/node-health_care_region_page' },
+        { type: 'null' },
+      ],
+    },
+    fieldOperatingStatusEmergInf: {
+      // Literally everywhere else, this would be a straight-up string
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js]: :joy:

File: src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    entityType: { enum: ['node'] },
+    entityBundle: { enum: ['vamc_operating_status_and_alerts'] },
+    title: { type: 'string' },
+    entityPublished: { type: 'boolean' },
+    entityMetatags: { $ref: 'MetaTags' },
+    entityUrl: { $ref: 'EntityUrl' },
+    fieldBannerAlert: {
+      type: 'array',
+      items: { $ref: 'transformed/node-full_width_banner_alert' },
+      // maxItems: 1 // Presumably? The template expects an array, though...
```
Comment (bkjohnson) [inline src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js]: So should this be uncommented?

**Review by bkjohnson — APPROVED** [review]
Looks good!  Some of those minor inconsistencies seemed annoying, especially how we have the links/uri/url pieces that are so similar but not _quite_ identical.

File: src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js (hunk 4 above)
Comment (cvalarida) [reply src/site/stages/build/process-cms-exports/schemas/transformed/node-vamc_operating_status_and_alerts.js]: :man_shrugging: Depends on how it's used in the template. I didn't know, so I left it open.

File: src/site/stages/build/process-cms-exports/schemas/transformed/node-full_width_banner_alert.js (hunk 2 above)
Comment (cvalarida) [reply src/site/stages/build/process-cms-exports/schemas/transformed/node-full_width_banner_alert.js]: Yeah, this'll be fleshed out when we write / merge in the transformer for this entity. For now, I just put this in here so it didn't fail the validation.

**Pull request merged**